### ISSUE TITLE
Auto-split line

### ISSUE THREAD

**Issue by ruiprieto** [franciscoaleixo/photogrammetry#47]
It would be useful to be able to auto-segment a line into pre-specified segment fractions (%). For example, after being drawn, I could select it and using a right click ask it to be split at 10, 50, and 73% of the total length creating three new lines with lengths of 10, 50 and 73% of the initial one. The justification is that sometimes measurements along a structure/body, are taken at pre-set intervals.

**Comment by franciscoaleixo** [comment]
Is it actually useful to define what % the points are? What would be a use case for this? I can see maybe being useful to split a line into 2, 3, 4, etc. but I don't especially see a use to define what % those points are. Because if they're not that useful, I could just split the line linearly (e.g. splitting a segment into 2 would be done at 50%, splitting into 3 would be done at 33% and 66%, etc.)

**Comment by franciscoaleixo** [comment]
I say this because I assume that if an user wants to split a segment it is because it needs further adjustment or "precision". I don't know how he could know the exact %

Edit: I didn't saw the justification properly. I see. I don't know how we could do it with a good user experience.. Would a dialog pop-up?

**Comment by ruiprieto** [comment]
Yes, that could work. Perhaps a right-click on the select tool (right click on this tool is inactive now). 

Then from a menu, the user would select 'Auto-split' and a pop-up opens where a series of values can be added.

Values can be %, or a fraction whichever is easiest to implement. 

If making it for several values at a time is difficult, then it could be just one value, as long as the original line is preserved.

The new line will have the same angular attributes of the original, but only a fraction of the dimension; Ideally it would also have a new colour, but that may be too much to ask
